src/pages/GraphiQL/GraphiQL.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC, Suspense, lazy } from 'react';
+import Loader from '../../components/Loader/Loader';
+import DocsModal from '../../components/GraphiQL/DocsModal';
+import { useAppSelector } from '../../hooks/useAppSelector';
+import { docsShown } from '../../store/slices/docsSlice';
+
+const GraphiQL: FC = () => {
+  const isDocsShown = useAppSelector(docsShown);
+
+  const LazyControlPanel = lazy(
+    () => import('../../components/GraphiQL/ControlPanel')
+  );
+  const LazyEditorsSection = lazy(
+    () => import('../../components/GraphiQL/EditorsSection')
+  );
+  const LazyResultsSection = lazy(
+    () => import('../../components/GraphiQL/ResultsSection')
+  );
```

Comment:
what is the purpose of lazy loading of these components?

Reply:
Were used to lazily load the component and reduce the original size of the bundle. But in this case it really was possible to wrap everything in one component, you are right +